src/app/pages/Search/Search.module.css

```diff
@@ -0,0 +1,26 @@
+.page {
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  background-color: var(--bg-primary);
+  color: var(--text-primary);
+  height: 100vh;
+}
+
+main {
+  overflow: auto;
+}
+
+.search {
+  width: 100vw;
```

Comment:
Is there a way to not set this property anywhere on lower leves than `#app` or the main grid for this page?
We might have to set a `max-width` for `TagGroup`